src/SubscriptionEvent.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace ChartMogul;
+
+use ChartMogul\Http\ClientInterface;
+use ChartMogul\Resource\AbstractResource;
+use ChartMogul\Service\AllTrait;
+use ChartMogul\Service\CreateTrait;
+use ChartMogul\Service\UpdateWithParamsTrait;
+use ChartMogul\Service\DestroyWithParamsTrait;
+use ChartMogul\Resource\Collection;
+
+class SubscriptionEvent extends AbstractResource
+{
+    use AllTrait;
+    use CreateTrait;
+    use UpdateWithParamsTrait;
+    use DestroyWithParamsTrait;
+
+    /**
+     * @ignore
+     */
+    public const RESOURCE_PATH = '/v1/subscription_events';
+    public const RESOURCE_ID = 'id';
```

Comment:
Correct me if I'm wrong but as far as I can see we're not going to use the `RESOURCE_ID` const anywhere